src/odemis/util/conversion.py

```diff
@@ -392,3 +394,78 @@ def ensure_tuple(v):
     else:
         return v
 
+def get_img_transformation_matrix(md):
+    """
+    Computes the 2D transformation matrix based on the given metadata.
+    md (dict str -> value): the metadata (of the DataArray) containing MD_PIXEL_SIZE 
+        and possibly also MD_ROTATION and MD_SHEAR.
+    return (numpy.matrix of 2,2 floats): the 2D transformation matrix
+    """
+
+    if model.MD_PIXEL_SIZE not in md:
+        raise ValueError("MD_PIXEL_SIZE must be set")
+    ps = md[model.MD_PIXEL_SIZE]
+    rotation = md.get(model.MD_ROTATION, 0.0)
+    shear = md.get(model.MD_SHEAR, 0.0)
+
+    # Y pixel coordinates goes down, but Y coordinates in world goes up
+    # The '-' before ps[1] is there to make this conversion
+    ps_mat = numpy.matrix([[ps[0], 0], [0, -ps[1]]])
+    cos, sin = numpy.cos(rotation), numpy.sin(rotation)
```

Comment:
No need to use `numpy` for computing cos and sin, just use `math`.
Also, don't call them `cos/sin` (too generic), but something like `rcos` and `rsin`.